src/v0/destinations/google_adwords_remarketing_lists/recordTransform.js

```diff
@@ -0,0 +1,173 @@
+/* eslint-disable no-const-assign */
+const lodash = require('lodash');
+const { InstrumentationError, getErrorRespEvents } = require('@rudderstack/integrations-lib');
+const {
+  getValueFromMessage,
+  getAccessToken,
+  constructPayload,
+  returnArrayOfSubarrays,
+  getSuccessRespEvents,
+  generateErrorObject,
+} = require('../../util');
+const { populateConsentFromConfig } = require('../../util/googleUtils');
+const { populateIdentifiers, responseBuilder } = require('./util');
+
+const { offlineDataJobsMapping, consentConfigMap } = require('./config');
+
+function getErrorMetaData(inputs, acceptedOperations) {
```

Comment:
should we create `recordUtils.js` & move these common operations which are needed across destinations ?

Reply:
common functions between record destinations are moved to `recordUtils.js`